reana_client/cli/workflow.py

```diff
@@ -539,10 +533,71 @@ def workflow_run(ctx, file, filenames, name, skip_validation,
                parameter=parameter)
 
 
+@click.command(
+    'delete',
+    help='Delete a workflow run.')
+@click.option(
+    '-A',
+    '--all',
+    'all_runs',
+    count=True,
+    help='Delete all runs of a given workflow.')
+@click.option(
+    '--hard',
+    'hard_delete',
+    count=True,
+    help='Completely remove workflow run data and workspace from REANA.')
+@click.option(
+    '-w',
+    '--workflow',
+    default=os.environ.get('REANA_WORKON', None),
+    callback=workflow_uuid_or_name,
+    help='Name and run number to be deleted. '
+         'Overrides value of REANA_WORKON.')
+@add_access_token_options
+@click.pass_context
+@with_api_client
+def workflow_delete(ctx, workflow, all_runs, hard_delete, access_token):
+    """Delete a workflow run given the workflow name and run number."""
+    logging.debug('command: {}'.format(ctx.command_path.replace(" ", ".")))
+    for p in ctx.params:
+        logging.debug('{param}: {value}'.format(param=p, value=ctx.params[p]))
+
+    if not access_token:
+        click.echo(
```

Comment:
Why not click.secho?

Reply:
`click.echo(click.style())` you'll see is in a lot of places in the codebase, we can change it everywhere in one go if we think this is important